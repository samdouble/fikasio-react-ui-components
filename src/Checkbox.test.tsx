import React from 'react';
import { render } from '@testing-library/react';
import { Checkbox } from '.';

describe('Checkbox', () => {
  it('Renders correctly', () => {
    const { baseElement } = render(<Checkbox />);
    expect(baseElement).toMatchSnapshot();
  });
});
