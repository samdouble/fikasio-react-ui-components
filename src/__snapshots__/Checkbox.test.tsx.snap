// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox Renders correctly 1`] = `
<body>
  <div>
    <div
      class="fikasio-checkbox fikasio-theme-dark"
      role="button"
      tabindex="0"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-square fa-lg "
        data-icon="square"
        data-prefix="fas"
        focusable="false"
        role="img"
        style="border: 1px solid #7e5b9a; color: rgb(255, 255, 255); width: 20px;"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 96C0 60.7 28.7 32 64 32H384c35.3 0 64 28.7 64 64V416c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V96z"
          fill="currentColor"
        />
      </svg>
    </div>
  </div>
</body>
`;
